lib/providers/models/explore_card_filters_model.dart

```diff
@@ -22,7 +23,10 @@ class ExploreCardFiltersModel extends ChangeNotifier {
 
   String? get selectedIndustry => _selectedIndustry;
   Set<String> get selectedUserTypes => _selectedUserTypes;
```

Comment:
The backend doesn't actually list the user types, but right now we just have two: "Mentor" and "Mentee". There isn't any plan in the scope to add more user types that can be searched for. (We do have `User.roles`, but that's different than the type of user. Mentor/mentee are not in that list of roles.)

Are these currently hardcoded somewhere here? I think having them in the front end is fine for now.

Reply:
Yeah, this is the only value that remained hardcoded: https://github.com/micromentor-team/mm-flutter-app/pull/139/files#diff-1c59c7884f660b111a9fb79c05076a83e4ad06417087d8bea5b2782e30af5f3aR10